### PR TITLE
docs(cli): add bench commands to readme

### PR DESCRIPTION
**Pull request by Red-Asuka**
### PR Checklist

If you have any questions, you can refer to the [Contributing Guide](https://github.com/emqx/MQTTX/blob/master/.github/CONTRIBUTING.md)

#### What is the current behavior?

Please describe the current behavior and link to a relevant issue.

#### Issue Number

Example: \#123

#### What is the new behavior?

Please describe the new behavior or provide screenshots.

#### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

#### Specific Instructions

Are there any specific instructions or things that should be known prior to review?

### CODE REVIEW

File: cli/README-CN.md [hunk 1]
```diff
@@ -118,6 +118,24 @@ Publish
...
+
+```shell
+mqttx bench sub -c 5000 -t bench/%i
+```
+
+Publish Benchmark
+
+```shell
+mqttx bench pub -c 5000 -t bench/%i
+```
```
Comment (ysfscream) [inline cli/README-CN.md:137]: All in one shell

Benchmark

```
# Connect Benchmark

mqttx bench conn -c 5000

# Sub

...
```

File: cli/README.md [hunk 2]
```diff
@@ -117,6 +117,24 @@ Publish
...
+```shell
+mqttx bench sub -c 5000 -t bench/%i
+```
+
+Publish Benchmark
+
+```shell
+mqttx bench pub -c 5000 -t bench/%i
+```
+
```
Comment (ysfscream) [inline cli/README.md:137]: Same here.

File: cli/README-CN.md [hunk 3]
```diff
@@ -118,6 +118,24 @@ Publish
...
+
```
Comment (ysfscream) [inline cli/README-CN.md:138]: All in one shell

Benchmark

```shell
# Connect

mqttx bench conn -c 5000

# Subscribe

mqttx bench sub -c 5000 -t bench/%i

...
```

File: cli/README-CN.md [hunk 4]
```diff
@@ -279,6 +293,53 @@ mqttx pub --help
...
+#### 连接性能测试
+
+```shell
+mqttx bench conn --help
+```
+
+| 参数                          | 描述                             |
+| ----------------------------- | -------------------------------- |
+| -c, --count <NUMBER>          | 连接数量，默认为 1000            |
+| -i, --interval <MILLISECONDS> | 创建连接的间隔时间，默认为 10ms  |
```
Comment (ysfscream) [inline cli/README-CN.md:309]: 描述新增：单位为毫秒

File: cli/README-CN.md [hunk 5]
```diff
@@ -279,6 +293,53 @@ mqttx pub --help
...
+#### 订阅性能测试
+
+```shell
+mqttx bench sub --help
+```
+
+| 参数                          | 描述                                                                 |
+| ----------------------------- | -------------------------------------------------------------------- |
+| -c, --count <NUMBER>          | 连接数量，默认为 1000                                                |
+| -i, --interval <MILLISECONDS> | 创建连接的间隔时间，默认为 10ms                                      |
```
Comment (ysfscream) [inline cli/README-CN.md:321]: 描述新增：单位为毫秒

File: cli/README-CN.md [hunk 6]
```diff
@@ -279,6 +293,53 @@ mqttx pub --help
...
+
+```shell
+mqttx bench pub --help
+```
+
+| 参数                                   | 描述                                                                 |
+| -------------------------------------- | -------------------------------------------------------------------- |
+| -c, --count <NUMBER>                   | 连接数量，默认为 1000                                                |
+| -i, --interval <MILLISECONDS>          | 创建连接的间隔时间，默认为 10ms                                      |
+| -im, --message-interval <MILLISECONDS> | 发布消息的间隔时间，默认为 1000ms                                    |
```
Comment (ysfscream) [inline cli/README-CN.md:336]: 描述新增：单位为毫秒

File: cli/README-CN.md [hunk 7]
```diff
@@ -279,6 +293,53 @@ mqttx pub --help
 | -Cup, --conn-user-properties <USERPROPERTIES...> | MQTT 5.0 的连接用户属性（例如，-Cup "name: mqttx cli"）      |
 | --help                                           | 展示 pub 命令的帮助信息                                      |
 
+### 性能测试
+
+性能测试命令与普通命令参数基本相同，以下仅会列出新增或有变化的参数
```
Comment (ysfscream) [inline cli/README-CN.md:298]:
```suggestion
性能测试命令与普通命令参数基本相同，以下仅会列出新增或有变化的参数
性能测试命令与普通命令参数基本相同，以下仅列出新增或有变化的参数
```

File: cli/README-CN.md (hunk 5 above)
Comment (Red-Asuka) [inline cli/README-CN.md:321]:
```suggestion
| -i, --interval <MILLISECONDS> | 创建连接的间隔时间，单位为毫秒，默认为 10ms                                      |
```

File: cli/README-CN.md [hunk 8]
```diff
@@ -279,6 +293,53 @@ mqttx pub --help
...
+#### 发布性能测试
+
+```shell
+mqttx bench pub --help
+```
+
+| 参数                                   | 描述                                                                 |
+| -------------------------------------- | -------------------------------------------------------------------- |
+| -c, --count <NUMBER>                   | 连接数量，默认为 1000                                                |
+| -i, --interval <MILLISECONDS>          | 创建连接的间隔时间，默认为 10ms                                      |
```
Comment (Red-Asuka) [inline cli/README-CN.md:335]:
```suggestion
| -i, --interval <MILLISECONDS>          | 创建连接的间隔时间，单位为毫秒，默认为 10ms                                      |
```

**Review by ysfscream — APPROVED** [review]

**Pull request merged**